web/app/themes/ppj/src/js/main.js

```diff
@@ -31,29 +10,44 @@ window.ppjNavTo = function(href, callback) {
   window.location = href;
 };
 
-window.addEventListener('load', function() {
+window.ppj.openNavMenu = function() {
+  document.getElementsByTagName('body')[0].style.overflow = 'hidden';
+  document.getElementsByClassName('header')[0].classList.add('header--nav-menu-open');
+};
+
+window.ppj.closeNavMenu = function() {
+  document.getElementsByTagName('body')[0].style.overflow = '';
+  document.getElementsByClassName('header')[0].classList.remove('header--nav-menu-open');
+};
+
+window.ppj.toggleAccordion = function() {
+  const event = event || window.event;
+  event.preventDefault();
+  event.stopPropagation();
+  const triggerElement = event.target || event.srcElement;
+  const className = 'accordion__list-element--open';
+  const el = triggerElement.closest('.accordion__list-element');
+  if (el.classList.contains(className)) {
+    el.classList.remove(className);
+  } else {
+    el.classList.add(className);
+    ga('send', 'event', 'Accordian', this.title, (this.open) ? 'open' : 'close', 7);
+  }
+};
 
-  if (document.querySelectorAll('#site-container').length > 0) {
+window._wq = window._wq || [];
```

Comment:
What is `window._wq` and what is it used for?

Reply:
It's a handle for Wistia's API. I've added a short comment.